composer.json

```diff
@@ -24,7 +24,7 @@
     }
   },
   "require": {
-    "php": ">=5.4",
+    "php": "^5.6 || ^7.1",
```

Comment:
maybe ^5.4 || 7.1 is better, as PrestaShop 1.7.3 will be the very last version to support php 5.4

Reply:
Well, in this case we have to rollback tests again 5.4 and 5.5.

Reply:
We don't support unmaintained PHP versions 'but' the minimal requirements of PrestaShop 1.7.3 is PHP 5.4. So the users must be able to install it on PHP 5.4 environnements *but* if bugs happens, it's up to them to fix it.

Reply:
> but if bugs happens, it's up to them to fix it.

Depends of where the bug is.